ecosystem/gstreamer_plugin/gst_mtl_st20p_rx.c

```diff
@@ -100,6 +100,11 @@ enum {
   PROP_MAX
 };
 
+typedef struct st20_rx_external_data {
+  GstBuffer* buf;
+  GstMapInfo dest_info;
+} st20_rx_external_data_t;
```

Comment:
This shoudn't be in the st20_rx 

first st20 suggest "legacy" API usage if this is meant for st20p -> pipeline mode then we should use st20p 

also st20 is a naming range reserved for the mtl, this is part of the ecosystem so 
`gst_st20p_rx` would be an appropriate prefix but i see that we are using the Gstreamer variables nameing convection 
so `GstSt20pRxExternalData` would be the goto